### PR TITLE
Added content to Workshops, Community Building, StudyGroup, Recruiting

### PR DESCRIPTION
**Pull request by arockenberger**

### CODE REVIEW

**Comment by arockenberger** [comment]
I added some stuff from the top up to but not including everything below "Organisation". Will not work more on this file today! Feel free to merge this pull request and make a branch for your own edits afterward!

File: reporting/report_carpentry-uio_2019.md [hunk 1]
```diff
@@ -1,29 +1,59 @@
...
 ## Community Building & Social Events
-- Christmas Party Dec 12
-- Onboarding sessions on Sep 2
-- coordination w/ NT (CodeRefinery/Carpentries Hub) and w/ MG (Digital Scholarship Centre)
-## StudyGroup
+
+- two onboarding sessions for new instructors and helpers were held on Sep 2
+- a Call for Helpers was sent to former workshop participants, new helpers and instructors were recruited (see [## Recruiting])
+- coordination with the newly appointed Regional Coordinator for the Nordic and Baltic Countries was established, with Oslo becoming a CodeRefinery and Carpentries Hub
+- coordination with the Digital Scholarship Centre (University Library) was established to serve as a virtual "home" for Carpentry@UiO workshops and activities
```
Comment (lexnederbragt) [inline reporting/report_carpentry-uio_2019.md]: We don't have a Digital Scholarship Centre yet at UiO but there is a process to establish one. Maybe rephrase a little? And explain what it is (going to be)?

File: reporting/report_carpentry-uio_2019.md [hunk 2]
```diff
@@ -1,29 +1,59 @@
...
+- End-of-the-Year Party on Dec 12
+
+### StudyGroup
+- the Carpentry@UiO community has taken initaitive for a StudyGroup in 2018. The aim was to create a forum for instructors, helpers, and other interested parties to meet and share expertise, inforamtion and experience related to teaching workshops. Since then, the StudyGroup has served as a forum for knowledge exchange and social interaction with meetings ca. every other week and an average of 5 active participants.
```
Comment (lexnederbragt) [inline reporting/report_carpentry-uio_2019.md]: initiative, information

File: reporting/report_carpentry-uio_2019.md [hunk 3]
```diff
@@ -1,29 +1,59 @@
...
 - overview events and topics, participants (number)
-- documentation on website
-- announcements of activities on mailing lists, social media
+- documentation on website (StudyGroup)[https://uio-carpentry.github.io/studyGroup/]
```
Comment (lexnederbragt) [inline reporting/report_carpentry-uio_2019.md]: link formatting (also elsewhere). Did you mean`[StudyGroup](https://uio-carpentry.github.io/studyGroup/)`?

**Comment by lexnederbragt** [comment]
Apologies for after-merge comments...

File: reporting/report_carpentry-uio_2019.md (hunk 1 above)
Comment (arockenberger) [reply reporting/report_carpentry-uio_2019.md]: Thanks! This really is a draft, but I needed to give NT access to it :)

File: reporting/report_carpentry-uio_2019.md (hunk 2 above)
Comment (arockenberger) [reply reporting/report_carpentry-uio_2019.md]: Thanks for spotting!
